### PR TITLE
zh-translation: docs/ops/deployment/architecture/index.md

### PR DESCRIPTION
**Pull request by orangegzx**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/ops/deployment/architecture/index.md [hunk 1]
```diff
@@ -40,48 +43,27 @@ Envoy 代理被部署为服务的 sidecar，在逻辑上为服务增加了 Envoy
...
-1. **Pilot** 将流量规则和配置派发给 Envoy 代理，来传达此次更改。
-
-这种松耦合允许 Istio 在 Kubernetes、Consul 或 Nomad 等多种环境中运行，同时维护相同的 operator 接口来进行流量管理。
-
-您可以使用 Istio 的[流量管理 API](/zh/docs/concepts/traffic-management/#introducing-Istio-traffic-management) 来指示 Pilot 优化 Envoy 配置，以便对服务网格中的流量进行更细粒度地控制。
+- 安全性和身份认证特性：执行安全性策略，并强制实行通过配置 API 定义的访问控制和速率限制。
+- 基于 WebAssembly 的可插拔扩展模型，允许通过自定义策略执行和生成网格流量的遥测。
 
-### Citadel{#citadel}
+### Istiod{#Istiod}
```
Comment (kebe7jun) [inline content/zh/docs/ops/deployment/architecture/index.md:57]: 英文 title 会自动转，不用手动加 ID。
```suggestion
### Istiod
```

File: content/zh/docs/ops/deployment/architecture/index.md [hunk 2]
```diff
@@ -40,48 +43,27 @@ Envoy 代理被部署为服务的 sidecar，在逻辑上为服务增加了 Envoy
...
 
-[Citadel](/zh/docs/concepts/security/) 通过内置的身份和证书管理，可以支持强大的服务到服务以及最终用户的身份验证。您可以使用 Citadel 来升级服务网格中的未加密流量。使用 Citadel，operator 可以执行基于服务身份的策略，而不是相对不稳定的 3 层或 4 层网络标识。从 0.5 版开始，您可以使用 [Istio 的授权特性](/zh/docs/concepts/security/#authorization)来控制谁可以访问您的服务。
+Istiod 提供服务发现、配置和证书管理。
 
-### Galley{#galley}
+Istiod 将控制流量行为的高级路由规则转换为 Envoy 特定的配置，并在运行时将其传播给 Sidecars。Pilot 提取了特定平台的服务发现机制，并将其综合为一种标准格式，任何符合 [Envoy API](https://www.envoyproxy.io/docs/envoy/latest/api/api) 的 Sidecar 都可以使用。
```
Comment (kebe7jun) [inline content/zh/docs/ops/deployment/architecture/index.md:61]:
```suggestion
Istiod 将控制流量行为的高级路由规则转换为 Envoy 特定的配置，并在运行时将其传播给 Sidecar。Pilot 提取了特定平台的服务发现机制，并将其综合为一种标准格式，任何符合 [Envoy API](https://www.envoyproxy.io/docs/envoy/latest/api/api) 的 Sidecar 都可以使用。
```

File: content/zh/docs/ops/deployment/architecture/index.md [hunk 3]
```diff
@@ -40,48 +43,27 @@ Envoy 代理被部署为服务的 sidecar，在逻辑上为服务增加了 Envoy
...
 
-Galley 是 Istio 的配置验证、提取、处理和分发组件。它负责将其余的 Istio 组件与从底层平台（例如 Kubernetes）获取用户配置的细节隔离开来。
+Istio 可以支持发现多种环境，如 Kubernetes 或 VM。
 
-## 设计目标{#design-goals}
+您可以使用 Istio [流量管理 API](/zh/docs/concepts/traffic-management/#introducing-istio-traffic-management) 来指示 Istiod 优化 Envoy 的配置，以便对服务网格中的流量进行更精细的控制。
```
Comment (kebe7jun) [inline content/zh/docs/ops/deployment/architecture/index.md:65]:
```suggestion
您可以使用 Istio [流量管理 API](/zh/docs/concepts/traffic-management/#introducing-istio-traffic-management) 让 Istiod 重新构造 Envoy 的配置，以便对服务网格中的流量进行更精细的控制。
```

**Review by rootsongjc — APPROVED** [review]
/LGTM

**Pull request merged**